cmd/minishift/cmd/image/image.go

```diff
@@ -20,11 +20,14 @@ import (
 	"github.com/spf13/cobra"
 )
 
+const (
+	noCachedImagesSpecified = "You need to either specify a list of images on the command line or set the persistent configuration property 'cache-images'."
+)
+
 var ImageCmd = &cobra.Command{
-	Use:    "image SUBCOMMAND [flags]",
-	Short:  "Exports and imports container images (experimental).",
-	Long:   "Exports and imports container images (experimental).",
-	Hidden: true,
+	Use:   "image SUBCOMMAND [flags]",
```

Comment:
I would suggest we add `Aliases: []string{"images"}` as in sub-commands we have referred to images e.g. 
```
$ minishift image config add --help
Adds the specified images to the list of configured images for import and export.
```

Reply:
I am ok with adding the plural as ana alias. I don't get your reasoning though. One is the command 'image' and one is multiple container images. No relation to the command there.